core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -694,6 +697,16 @@ public ServerBuilder meterRegistry(MeterRegistry meterRegistry) {
         return this;
     }
 
+    /**
+     * Sets a global naming rule for the name of services.
+     *
+     * @see RequestOnlyLog#serviceName()
+     */
+    public ServerBuilder serviceNaming(ServiceNamingRule serviceNaming) {
```

Comment:
I guess we should rename this to `defaultServiceNaming` once we make the changes I suggested.